btp-core/src/test/java/org/interledger/btp/BtpMessageTest.java

```diff
@@ -20,13 +20,17 @@
  * =========================LICENSE_END==================================
  */
 
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 import org.interledger.btp.BtpSubProtocol.ContentType;
 
 import org.junit.Test;
 
 import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
```

Comment:
These 4 new imports are unused and unnecessary.

Reply:
Good catch! Addressed it in the latest amended commit.